app/models/user.rb

```diff
@@ -25,7 +25,7 @@ def can_edit_site?(site_to_edit)
         own_organisation == site_to_edit.organisation ||
         site_to_edit.organisation.parent_organisations.include?(own_organisation) ||
         site_to_edit.extra_organisations.include?(own_organisation) &&
-        site_to_edit.global_type.blank?
+          site_to_edit.global_type.blank?
```

Comment:
ok, so the reason for this is that what we have is: 

```ruby
a || b || c || d && e
```

What this actually means is:

```ruby
a || b || c || (d && e)
```

So rubocop sees the `d && e` part as a new expression that should be indented separately.  I wonder if that's actually what was intended, or if we meant `(a || b || c || d) && e`.